java/build.gradle

```diff
@@ -17,7 +17,7 @@ repositories {
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
-def grpcVersion = '1.37.0' // CURRENT_GRPC_VERSION
+def grpcVersion = '1.40.0' // CURRENT_GRPC_VERSION
```

Comment:
Also update the version for cpp? Is it available now?

Reply:
It's not available yet, but should be coming out soon. Updated, but will not merge until it's available.